supplementary_style_guide/glossary_terms_conventions/product_conventions/eap-conventions/eap-terms-and-usage--m.adoc

```diff
@@ -0,0 +1,209 @@
+= M
+
+[discrete]
+=== mail
+
+*Description* The JBoss EAP subsystem that is used to configure mail services for applications deployed to JBoss EAP.
+
+*Use it?* Yes
+
+*Incorrect forms* Mail
+
+*See* Mail
+
+[discrete]
+=== Mail
+
+*Description* Heading format for the mail subsystem.
+
+*Use it?* Yes
+
+*Incorrect forms* mail
+
+*See* mail
+
+[discrete]
+=== managed domain
+
+*Description* A group of JBoss EAP instances managed from a single control point.
+
+*Use it?* Yes
+
+*Incorrect forms* domain mode
+
+*See*
+
+[discrete]
+=== management console
+
+*Description* The preferred term for the web-based JBoss EAP management console.  Do not capitalize "management" unless it starts a sentence.
```

Comment:
There are two spaces after the first sentence.

Reply:
This is now fixed based on updates to the jboss-eap.adoc file in https://github.com/redhat-documentation/supplementary-style-guide/pull/97.